packages/widgetbook_core/test/src/navigation_tree/bloc/navigation_bloc_test.dart

```diff
@@ -158,7 +158,9 @@ void main() {
         filteredNodes: nodes,
       ),
       act: (bloc) => bloc.add(
-        const SelectNavigationNodeByPath(path: 'package/component/use-case-1'),
+        const SelectNavigationNodeByPath(
+          path: '/?path=package/component/use-case-1',
+        ),
```

Comment:
The path provided is an unencoded one. This basically makes the test pass, however, we need to invest more time to properly adjust the API.